game-core/src/main/java/games/strategy/engine/framework/LocalPlayers.java

```diff
@@ -19,11 +19,9 @@ public LocalPlayers(final Set<IGamePlayer> localPlayers) {
   }
 
   public boolean playing(final PlayerID id) {
-    return id != null
-        && localPlayers.stream()
-            .anyMatch(gamePlayer -> isGamePlayerWithPlayerId(gamePlayer, id));
+    return id != null && localPlayers.stream().anyMatch(gamePlayer -> isGamePlayerWithPlayerId(gamePlayer, id));
```

Comment:
There should be a line break after `localPlayers.stream()`

Reply:
Will fix tomorrow unless you Want to push a chance to this branch. 
IMO a line break is not needed, because everything fits in one line.

Reply:
> IMO a line break is not needed, because everything fits in one line.

 : )

while everything may fit, it's not the best idea always. Example, let's say we had a 160 line limit, we would not want many streaming operations on the same line.

The guiding principle here is "one logical operation per line".
Doing something and starting a stream is an operation.
Doing an any match is another operation, so on it's own line. 

This is why we format like:
```
stream()
  .filter(...)
  .map(...)
  .filter(...)
  .findAny(...)
```
Instead of doing just what fits on a single line:
```
stream().filter(...).map(...)
  .filter(...).findAny(...)
```

The latter would be like writing (many operations all on the same line):
``` for ( String s : strings) { s += "abc"; foo(s); if( s != null) { return true }} ```

So in one way, "dot operators" are almost as good as an open curly braces when it comes to line breaking.

Reply:
I don't have access to an IDE to make sure I don't introduce a checkstyle; I can potentially fix it in a few hours later tonigth for you though @RoiEXLab .